src/webgpu/api/operation/debug/debug_marker.spec.ts

```diff
@@ -0,0 +1,32 @@
+export const description = `
+Test operations of pushDebugGroup, popDebugGroup, and insertDebugMarker.
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+import { kEncoderTypes } from '../../../util/command_buffer_maker.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('no_failure_without_debug_tool_attached')
```

Comment:
I write this test referring to the dawn end2end test[1]. However, I'm not fully sure if we need to add this test to ensure the operation since there are similar tests regarding this in some validation tests. wdyt?

[1] https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/tests/end2end/DebugMarkerTests.cpp;l=22?q=NoFailureWithoutDebugToolAttached&ss=chromium

Reply:
As long as we have *some* test which calls the debug commands without expecting an error, I think that's good enough. I think the tests in #1921 will do this.

Reply:
And we also have `api/validation/encoding/cmds/debug.spec.ts`

Reply:
ok, then let me abandon this PR. Thanks.